src/libraries/System.Private.Xml.Linq/tests/Schema/ExtensionTests.cs

```diff
@@ -27,397 +27,397 @@ namespace CoreXml.Test.XLinq
+            int beforeNoOfAttributes = 0;
+            while (elementsEnumerator.MoveNext())
+            {
+                    beforeNoOfElements++;
+                    if (!elementsEnumerator.Current.HasAttributes)
+                        continue;
+
+                        IEnumerable<XAttribute> attributes = elementsEnumerator.Current.Attributes();
+                        IEnumerator<XAttribute> attributesEnumerator = attributes.GetEnumerator();
+                        while (attributesEnumerator.MoveNext())
+                            beforeNoOfAttributes++;
+            }
+
+            // Populate XML with PSVI.
+            validationSucceded = true;
+            ExtensionsClass.Validate(xmlDocument, schemaSet,
+                new ValidationEventHandler(TestValidationHandler), true);
+            Assert.True(validationSucceded);
+
+            // No. of elements after validation.
+            elements = xmlDocument.Elements();
+            elementsEnumerator = elements.GetEnumerator();
+            int afterNoOfElements = 0;
+            int afterNoOfAttributes = 0;
+            while (elementsEnumerator.MoveNext())
+            {
+                afterNoOfElements++;
+                if (!elementsEnumerator.Current.HasAttributes)
+                    continue;
+
+                    IEnumerable<XAttribute> attributes = elementsEnumerator.Current.Attributes();
+                    IEnumerator<XAttribute> attributesEnumerator = attributes.GetEnumerator();
+                    while (attributesEnumerator.MoveNext())
+                        afterNoOfAttributes++;
+            }
+
+            Assert.True(afterNoOfAttributes >= beforeNoOfAttributes, "XDocumentAddSchemaInfoValidate, wrong newAttributes value.");
+            Assert.True(afterNoOfElements >= beforeNoOfElements, "XDocumentAddSchemaInfoValidate, wrong newElements value.");
+        }
+
+        /*
+         * test xml validation without populating the XML tree with
+         * the post-schema-validation infoset(PSVI).
+         */
+        [Fact]
+        public void XDocumentNoSchemaInfoValidate()
+        {
+            // No. of elements before validation.
+            IEnumerable<XElement> elements = xmlDocument.Elements();
+            IEnumerator<XElement> elementsEnumerator = elements.GetEnumerator();
+            int beforeNoOfElements = 0;
+            int beforeNoOfAttributes = 0;
+            while (elementsEnumerator.MoveNext())
+            {
+                beforeNoOfElements++;
+                if (!elementsEnumerator.Current.HasAttributes)
+                    continue;
+
+                    IEnumerable<XAttribute> attributes = elementsEnumerator.Current.Attributes();
+                    IEnumerator<XAttribute> attributesEnumerator = attributes.GetEnumerator();
+                    while (attributesEnumerator.MoveNext())
+                        beforeNoOfAttributes++;
+            }
+
+            // Don't populate XML with PSVI.
```

Comment:
I think adding a leading capital and trailing period is a low value change -- might be harder to read quickly for some people even. As I mentioned, I don't see it in the style guidelines.  Also we don't do it consistently, so if you start applying this rule, it will create changes everywhere.

I would rather avoid these kinds of changes, but if you do them here, I would at least rather not try to do them everywhere.

Reply:
I agree.

Reply:
Also, it seems wierd to have a period after something that isn't a sentence, eg.
`            // No. of elements after validation.`

My preference would be to revert all the leading caps and trailing period. It's fine to have caps period in an extended comment, eg.,

```
// This is some extended comment talking about stuff that starts with caps and ends with period. Here is some more interesting commentage that has the same properties. This however as the last sentence, may or may not end in a period and typically won't.
```

Reply:
It's fine to open an issue to start discussion if you feel passionately about adding to the style guidelines.

Reply:
@danmosemsft, so I should revert all comment changes in this PR, right ?